### PR TITLE
Update build instructions for RHEL 8 distribution

### PR DESCRIPTION
**Pull request by bartwensley**
Updated build instructions for RHEL 8 distribution, based on
recent work done using RHEL 8.5 as the build system.

Signed-off-by: Bart Wensley <bwensley@redhat.com>

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Updated build instructions for RHEL 8 distribution, based on recent work done using RHEL 8.5 as the build system.

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by haircommander** [comment]
/approve

LGTM, thanks @bartwensley !

@cri-o/cri-o-maintainers PTAL

**Comment by bartwensley** [comment]
Thanks @haircommander for the quick review. I updated the commit with a missing "-" to fix a bulleted list.

File: install.md [hunk 1]
```diff
@@ -236,11 +237,10 @@ subscription-manager repos --enable=codeready-builder-for-rhel-8-x86_64-rpms
 Follow the guide below to subscribe to the repositories if not already subscribed:\
 https://access.redhat.com/solutions/265523
 
-This requires go version 1.12 or greater:
-```
-yum module -y install go-toolset
-```
+This requires go version 1.18 or greater. Follow these instructions to install go: \
```
Comment (TomSweeneyRedHat) [inline install.md:240]:
```suggestion
This requires Go version 1.18 or greater. Follow these instructions to install go: \
```

Comment (bartwensley) [reply install.md:240]: Done

**Comment by bartwensley** [comment]
I re-added the reviewers. Somehow they were removed when I replied to a comment.

**Comment by haircommander** [comment]
note: test failures aren't related to this PR, we're in the midst of a migration

**Comment by haircommander** [comment]
/lgtm

thanks!

**Review by saschagrunert — APPROVED** [review]
/retest-required

**Pull request merged**